jetty-start/src/test/java/org/eclipse/jetty/start/MainTest.java

```diff
@@ -146,13 +146,11 @@ public void testWithCommandLine() throws Exception
         assertThat("Extra Jar exists: " + extraJar, Files.exists(extraJar), is(true));
         assertThat("Extra Dir exists: " + extraDir, Files.exists(extraDir), is(true));
 
-        StringBuilder lib = new StringBuilder();
-        lib.append("--lib=");
-        lib.append(extraJar.toString());
-        lib.append(File.pathSeparator);
-        lib.append(extraDir.toString());
-
-        cmdLineArgs.add(lib.toString());
+        String lib = "--lib=" +
+            extraJar.toString() +
+            File.pathSeparator +
+            extraDir.toString();
+        cmdLineArgs.add(lib);
```

Comment:
Avoid even the remote possibility of a NPE with
```suggestion
        String lib = "--lib=" + extraJar + File.pathSeparator + extraDir;
        cmdLineArgs.add(lib);
```